run_travis.sh

```diff
@@ -6,6 +6,7 @@ set -ex
 echo "running $RUNTEST tests"
 if [ "$RUNTEST" == "frontend" ]; then
     source $HOME/.nvm/nvm.sh
+    npm install --production
```

Comment:
So this file is really for just running processes, we have `install_travis.hs` which runs `frontend.sh test`.

Reply:
Do you see what I'm missing [here](https://travis-ci.org/cfpb/cfgov-refresh/jobs/351095910)? Travis was failing on the app dependencies not being installed and this fixed it. Should I put this in `frontend.sh test`?

Reply:
Also, how are the Travis scripts called? Are the filenames magic names, or could we name them `travis_install.sh` and `travis_run.sh` so they collect together in the file list? (Or can we combine them?? They're both run when Travis is run, no?)

Reply:
If you check in travis.yml you can see how things work.

Reply:
>>> Do you see what I'm missing here?

looking into this, not sure why it's referencing `const unFormatUSD = require( '../../node_modules/unformat-usd' )` in `dom-values.js`

Reply:
I would try just removing the `'../../node_modules/`, I'm not sure why it's referenced that way if it's a dependency.